viewer/packages/sector-parser/app/testMaterials.ts

```diff
@@ -0,0 +1,276 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+import * as THREE from 'three';
+
+// TODO: Fix dependencies such that test app doesn't depend on core internals
+import matCapTextureImage from '../../../core/src/datamodels/cad/rendering/matCapTextureData';
+import { sectorShaders } from '../../../core/src/datamodels/cad/rendering/shaders';
+
+const matCapTexture = new THREE.Texture(matCapTextureImage);
+matCapTexture.needsUpdate = true;
+
+export function createInstancedMeshMaterial(): THREE.ShaderMaterial {
```

Comment:
Not something we should fix now probably, but it would be nice if the material definitions was shared with Reveal and the test app. Doable?

Reply:
Absolutely, only reason this doesn't work like this now is that the place / package where these material definitions should be doesn't exist yet.